sky/exceptions.py

```diff
@@ -46,6 +46,14 @@ def with_failover_history(
         return self
 
 
+class InvalidConfigs(Exception):
+    """Raised when invalid configurations are provided for a given cloud.
+
+    The caller should handle this exception and skip the cloud.
```

Comment:
Not intuitive to say a caller of an Exception class

Reply:
Removed the sentence for now. Thanks!